packages/templates/src/cli.js

```diff
@@ -0,0 +1,153 @@
+          `div#template-grid-item-${gridItemId} div[data-testid="card-action-container"]`
+        )
+        .getAttribute('data-template-slug');
+    }, currentTemplate);
+    // Prep a directory for the template screenshots
+    fs.mkdirSync(`${screenshotsPath}${templateName}`, { recursive: true });
+
+    console.log(`Getting screenshots for ${templateName}`);
+
+    // Use the template, this will open up the editor
+    await page.click(
+      `div#template-grid-item-${currentTemplate} button[aria-label="Use this template"]`
+    );
+
+    // Load the template in the editor to get story id to render preview
+    await page.waitForNavigation({ waitUntil: 'networkidle2' });
+    const editorUrl = await page.url(); // 'http://localhost:8899/wp-admin/post.php?post=1163&action=edit#page=%252217fa9c85-89c7-47d0-a7a8-29f6ef56e161%2522'
+    const storyId = editorUrl.split('post=')[1].split('&')[0];
+
+    const previewUrl = `http://localhost:8899/?post_type=web-story&p=${storyId}&preview=true`;
+
+    // Need to click preview for the story to save and be loadable in a second window.
+    await page.waitForSelector('aria/Preview');
+    await page.click('aria/Preview');
+
+    const pagePreview = await browser.newPage();
+    await pagePreview.setViewport({
+      width: 1600,
+      height: 800,
+      deviceScaleFactor: 2,
+    });
+    // set prefers-reduced-motion to get story without animations so screenshots are complete page views
+    await pagePreview.emulateMediaFeatures([
+      { name: 'prefers-reduced-motion', value: 'reduce' },
+    ]);
+    await pagePreview.evaluate(
+      () => matchMedia('(prefers-reduced-motion: reduce)').matches
+    );
+
+    await pagePreview.goto(previewUrl);
+    // Ensure that the page's ready before looping
+    await pagePreview.waitForSelector('aria/Next page');
+
+    const totalPages = await pagePreview.evaluate(() => {
+      return document.querySelectorAll('amp-story-page').length;
+    });
+
+    for (let currentPage = 1; currentPage < totalPages; currentPage++) {
+      const templatePageSafeArea = await pagePreview.$(
+        `amp-story-page:nth-child(${currentPage}) .page-safe-area`
+      );
+      // Speed up animations, this in tandem with emulating reduced motion will account for fade in content
+      await pagePreview._client.send('Animation.setPlaybackRate', {
+        playbackRate: 2,
+      });
+
+      await templatePageSafeArea.screenshot({
+        path: `${screenshotsPath}${templateName}/${currentPage}.png`,
+      });
+
+      if (currentPage !== totalPages - 1) {
+        await pagePreview.click('aria/Next page');
+      }
+    }
+
+    // Close the preview
+    await pagePreview.close();
+
+    // Use the original browser page to go back to the template gallery for the next template.
+    await page.goto(
+      'http://localhost:8899/wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/templates-gallery'
+    );
+  }
+
+  process.stdout.write('.');
```

Comment:
Was that suppose to be within the loop to print a dot for every template? 🤔

Reply:
I liked it at the end - but it's kinda arbitrary. I'm printing out what template's getting worked on as it is. I think I'll delete it.

<img width="571" alt="Screen Shot 2021-08-18 at 8 48 25 AM" src="https://user-images.githubusercontent.com/10720454/129929992-4374877b-0680-4a57-9f5c-717a73a10099.png">